### ISSUE TITLE
Question: SWC module transpilation required for nullish coalescing operator 

### ISSUE THREAD

**Issue by simonhrogers** [coreyward/sanity-image#26]
Hey Corey – thanks so much for your work on this package. 

It took me a little while to debug this but if you’re targeting Safari 12/13 like Next itself is designed to cover, then adding module transpilation is necessary e.g. `transpilePackages: ['sanity-image']` inside your `next.config`. This is because the nullish coalescing operator is not supported by these browsers and SWC doesn’t transpile packages by default. 

I was wondering what you think about this – should the docs have a "gotchas" section which explains this?

**Comment by coreyward** [comment]
Hey @simonhrogers, thanks for the report! It looks like the nullish coalescing operator is supported by iOS Safari 13.4+ and desktop Safari 13.1+, each of which were released over 3 years ago, but targeting ES2019 (prior to `??` support) doesn't meaningfully impact the ESM output in either size or execution time, so I've opted to just update the build target to that (bf829ef4955f003ceff842fd61dc33ff156b5a47).

This change is out as v0.1.5, so upgrading should remove the need to have SWC perform transpilation.
